x/stakeibc/keeper/unbonding_records_initiate_all_unbondings_test.go

```diff
@@ -109,49 +109,51 @@ func (s *KeeperTestSuite) SetupInitiateAllHostZoneUnbondings() InitiateAllHostZo
 }
 
 func (s *KeeperTestSuite) TestInitiateAllHostZoneUnbondings_Successful() {
-	// tests that we can successful initiate a host zone unbonding for ATOM and OSMO
+	// tests that we can successful initiate a host zone unbonding for GAIA and OSMO
 	s.SetupInitiateAllHostZoneUnbondings()
-	success, successful_unbondings, failed_unbondings := s.App.StakeibcKeeper.InitiateAllHostZoneUnbondings(s.Ctx, 12)
-	s.Require().True(success, "initiating unbondings returns true")
-	s.Require().Len(successful_unbondings, 2, "initiating unbondings returns 2 successful unbondings")
-	s.Require().Len(failed_unbondings, 0, "initiating unbondings returns 0 failed unbondings")
+	s.App.StakeibcKeeper.InitiateAllHostZoneUnbondings(s.Ctx, 12)
+
+	// An event should be emitted for each if they were successful
+	s.CheckEventValueEmitted(types.EventTypeUndelegation, types.AttributeKeyHostZone, HostChainId)
```

Comment:
lines 117-170: why are we checking event emissions in unit tests rather than checking changes in underlying state?

Reply:
messaged you on slack to explain in detail, but TLDR is we don't need to check the nitty gritty in this test since that's covered by all the `TestUnbondFromHostZone` tests. All we need to do is make sure these didn't fail, but since no error is throw, we can verify that with events